CHANGELOG.rst

```diff
@@ -16,6 +16,11 @@ Backward-incompatible changes:
   The package meta data should ensure that you keep getting 20.1.0 on those versions.
   `#244 <https://github.com/hynek/structlog/pull/244>`_
 
+- ``structlog`` is now fully typed.
```

Comment:
```suggestion
- ``structlog`` is now fully type-annotated.
```

This is a tiny nitpick, but Python is *always* (strongly) typed; the change here is to add static type annotations.  Pedantic, but the conflation of strong typing with static typing really irritates me as a fan of strongly and dynamically typed Python!